infra-as-code/bicep/modules/policy/assignments/alzDefaults/alzDefaultPolicyAssignments.bicep

```diff
@@ -505,6 +505,13 @@ var varPrivateDnsZonesFinalResourceIds = {
   azureCognitiveSearchPrivateDnsZoneId: '${varPrivateDnsZonesBaseResourceId}privatelink.search.windows.net'
 }
 
+var varPolicyAssignmentScopeName = '${parTopLevelManagementGroupPrefix}${parTopLevelManagementGroupSuffix}'
+var varPolicyExemptionConfidentialOnlineManagementGroup = '${parTopLevelManagementGroupPrefix}-landingzones-confidential-online${parTopLevelManagementGroupSuffix}'
+var varPolicyExemptionConfidentialCorpManagementGroup = '${parTopLevelManagementGroupPrefix}-landingzones-confidential-corp${parTopLevelManagementGroupSuffix}'
+
+var varSlzGlobalLibDef = loadJsonContent('../lib/policy_assignments/policy_assignment_es_enforce_sovereignty_baseline_global.tmpl.json')
```

Comment:
Can we replace this variable with varPolicyAssignmentEnforceSovereignGlobal which already exists.

Reply:
Done